multihost_test/bz_automation/script/pamtest.c

```diff
@@ -0,0 +1,140 @@
+#include <security/pam_appl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <pwd.h>
+
+#define TEST_USER "testUser"
+#define TEST_PASSWORD "TestPassword"
+
+/* Define our service (could really be anything) and then our conversation
+   function. */
+const char *service="pamtest"; 
+extern int converse(int, const struct pam_message **, struct pam_response **, 
+        void *);
+
+int main(int argc, char *argv[])
+{
+        int rval;
```

Comment:
In sssd we are using 4 lines of indention and I think it would be nice if you could follow this approach, if not, I fear that you will make the code difficult to read.

Reply:
Fixed